src/main/java/xyz/vegaone/easytrackingv3/service/UserService.java

```diff
@@ -13,6 +14,8 @@
 import xyz.vegaone.easytrackingv3.repo.UserRepo;
 import xyz.vegaone.easytrackingv3.util.MapperUtil;
 
+import javax.persistence.PersistenceContext;
```

Comment:
Three imports were added that are not used. Could you please remove them? 
I usually do a CTRL+O to optimise (includes remove of unused) imports.